docs/reference/getting-started.asciidoc

```diff
@@ -31,156 +31,158 @@ trial of Elasticsearch Service] in the cloud.
 
 [[getting-started-install]]
 == Get {es} up and running
-To take {es} for a test drive, you can create a one-click cloud deployment on
-the https://www.elastic.co/cloud/elasticsearch-service/signup[Elasticsearch
-Service], or set up a basic {es} cluster on your own
-<<run-elasticsearch-linux,Linux>>,
-macOS, or
-<<run-elasticsearch-win,Windows>> machine.
 
-[float]
-[[run-elasticsearch-linux]]
-=== Run {es} on Linux
+To take {es} for a test drive, you can create a one-click cloud deployment
+on the https://www.elastic.co/cloud/elasticsearch-service/signup[Elasticsearch Service],
+or <<run-elasticsearch-local, set up a basic {es} cluster>> on your own Linux,
+macOS, or Windows machine.
+
```

Comment:
We may want to note here that we're creating a three-node cluster with a very brief explanation of why that's beneficial.

Reply:
Fixed.